crates/biome_js_analyze/tests/specs/correctness/noUnusedImports/invalid-import-namespace.ts.snap

```diff
@@ -1,6 +1,7 @@
 ---
 source: crates/biome_js_analyze/tests/spec_tests.rs
 expression: invalid-import-namespace.ts
+snapshot_kind: text
```

Comment:
Is this new? Which version of `cargo-insta` are you using?

Reply:
The latest one: cargo-insta 1.41.1